pulsar-metadata/src/test/java/org/apache/pulsar/metadata/MetadataCacheTest.java

```diff
@@ -324,13 +324,14 @@ public void insertionOutsideCache(String provider, Supplier<String> urlSupplier)
     }
 
     @Test(dataProvider = "impl")
-    public void insertionOutsideCacheWithGenericType(String provider, Supplier<String> urlSupplier) throws Exception {
+    public void updateOutsideCacheWithGenericType(String provider, Supplier<String> urlSupplier) throws Exception {
```

Comment:
why are you changing an existing test ?
can we add a new test ?

how are you verifying your change ?

Reply:
> Changed the test to `updateOutsideCacheWithGenericType` for now.
> 
> I think we can implement `insertionOutsideCacheWithGenericType` by make a path pattern registration or add a method like `accept(path)` for each MetadataStore instance, which implies the metadata store could deserialization this value, and the path indeed should be cached in this metadata store.

Please have a look at the conversation above with merlimat.

I am actually changing the feature for now, for a quick fix of the warning logs.
More complex solution is need to achieve the goal of **insertionOutsideCacheWithGenericType**.

Reply:
> how are you verifying your change ?

This test is modified to verify my change.